### PR TITLE
implement queries for getting reviews

### PR DESCRIPTION
**Pull request by impemberton**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -70,6 +70,128 @@ describe("/api/reviews", () => {
...
+              created_at: expect.any(String),
+            })
+          );
+        });
+        expect(res.body).toBeSortedBy("designer", {
+          ascending: true,
+        });
+      });
+  });
+  test("GET - 200 an invalid category returns an empty array", () => {
```
Comment (callumu21) [inline __tests__/app.test.js]: Be careful - we have two cases here. A valid category in the database with no associated reviews should return an empty array and 200. What about something that isn't in the database altogether?

File: output.txt [hunk 2]
```diff
@@ -1,5 +1,17 @@
-   owner    | review_id |       title       |     category     |   designer   |   owner    |          review_body           |                              review_img_url                               |       created_at        | votes | comment_count
```
Comment (callumu21) [inline output.txt:1]: Ideally, this should be .gitignored so that you only have it locally.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Just missing a test case that seems to have been mixed up with a different scenario. See my comment for advice 👍

**Comment by pkaodev** [comment]
Looks good, merge away!

**Pull request merged**